src/KingSystem/ActorSystem/actBaseProcUnit.cpp

```diff
@@ -73,7 +75,8 @@ void BaseProcUnit::unlinkProc(BaseProc* proc) {
 
 // NON_MATCHING
 void BaseProcUnit::cleanUp(BaseProc* proc, bool set_flag_5) {
-    BaseProcHandle* v6;  // x22
+    (void)proc;  // unused warning
```

Comment:
You can fix the unused warning by removing the `proc` name for the variable, which makes it clear that the parameter is unused.